packages/flutter_app_packager/lib/src/makers/rpm/make_rpm_config.dart

```diff
@@ -105,6 +105,10 @@ class MakeRPMConfig extends MakeConfig {
 
   @override
   Map<String, dynamic> toJson() {
+    final currentDate = DateTime.now();
+    DateTime now = DateTime.now();
```

Comment:
Possibly unnecessary duplicate variable? You could just use `currentDate`

Reply:
(or `now` even simpler;

As for the cp error, it seems it just needs the `../` relative path adding to the cp commands; so that ll131-143 would read
```
'%install': [
            'mkdir -p %{buildroot}%{_bindir}',
            'mkdir -p %{buildroot}%{_datadir}/%{name}',
            'mkdir -p %{buildroot}%{_datadir}/applications',
            'mkdir -p %{buildroot}%{_datadir}/metainfo',
            'mkdir -p %{buildroot}%{_datadir}/pixmaps',
            'cp -r ../%{name}/* %{buildroot}%{_datadir}/%{name}',
            'ln -s ../%{_datadir}/%{name}/%{name} %{buildroot}%{_bindir}/%{name}',
            'cp -r ../%{name}.desktop %{buildroot}%{_datadir}/applications',
            'cp -r ../%{name}.png %{buildroot}%{_datadir}/pixmaps',
            'cp -r ../%{name}*.xml %{buildroot}%{_datadir}/metainfo || :',
            'update-mime-database %{_datadir}/mime &> /dev/null || :',
            '%undefine __brp_add_determinism',
          ].join('\n'),
```
 (including the `%undefine` that hangs); maybe you could please test whether that is a complete resolution of the regression?